app/src/endpoints.c

```diff
@@ -23,8 +23,12 @@
 #include <zephyr/logging/log.h>
 LOG_MODULE_DECLARE(zmk, CONFIG_ZMK_LOG_LEVEL);
 
+#if IS_ENABLED(CONFIG_ZMK_ENDPOINT_DISABLE_FALLBACK)
+#define DEFAULT_TRANSPORT ZMK_TRANSPORT_NONE
```

Comment:
Unless I'm missing somewhere that this gets used, this is unused when CONFIG_ZMK_ENDPOINT_DISABLE_FALLBACK is enabled, so it doesn't need to be defined.

Reply:
Good point, it's been removed